### ISSUE TITLE
Stats calculations are off

### ISSUE THREAD

**Issue by bugzilla-to-github** [mozilla/pontoon#2040]
*This issue was created automatically by a [script](https://github.com/mathjazz/bugzilla2github/).*
## [Bug 1310668](https://bugzilla.mozilla.org/show_bug.cgi?id=1310668)
Bug Reporter: @mathjazz
CC:   dian.ina@gmail.com, @flodolo, @lliehu, @TheoChevalier, @yfdyh000
See also:     https://bugzilla.mozilla.org/show_bug.cgi?id=1381852

Stats do not always reflect the actual state.

Multiple inconsistencies occur in the data model from time to time. Here's more specifics for developers:
1. TranslatedResource: translated + suggested + fuzzy > total
2. TranslatedResource & Resource: total_strings missmatch
3. ProjectLocale within the same project: total missmatch

**Comment by flodolo** [comment]
Should this be called fixed at this point?

**Comment by mathjazz** [comment]
We still need to run calculate stats manually from time to time. Unless that's only caused by the "IntegrityError" issues that are already tracked separately?

**Comment by flodolo** [comment]
I've seen the number going negative if I translate in the middle of a sync, but it always fixes itself.

Before #4036 we had to recalculate the stats to unblock approval/submission, but that's not the case anymore.  We have #2284 left, but AFAIK that's not affecting stats.

In general, I don't think the initial premise is still valid (Multiple inconsistencies occur in the data model from time to time.).

**Comment by mathjazz** [comment]
OK, let's close and track individual issues we may hit separately.
